### ISSUE TITLE
`incorrect-decorator-order` (`RUF074`) - false positive in `.pyi` files

### ISSUE THREAD

**Issue by DetachHead** [astral-sh/ruff#25377]
### Summary

```py
# foo.pyi
class Foo:
    @abstractmethod # error: RUF074
    @property
    def foo(self) -> int: ...
```

since this rule is intended to prevent runtime errors, it should not be reported in `.pyi` files which are never executed at runtime

### Version

0.15.4

**Comment by DetachHead** [comment]
i guess on the other hand it doesn't hurt to always enforce the correct order for consistency

**Comment by MichaReiser** [comment]
I think the correct order is beneficial in case where someone wants to extend `Foo` and the LSP completes `foo`'s definition based on the `.pyi` file

**Comment by DetachHead** [comment]
true. i'll close this then, thinking about it more i would rather the rule be applied everywhere for consistency
